README.md

```diff
@@ -1,33 +1,349 @@
-# bioio
+# BioIO
```

Comment:
Should migration document be here in the README or as a top level document in the documentation website?

Reply:
My vote would be to put it around line 9 with the brief description of the package.

Reply:
Added to both kinda? Not exactly sure what to say in this document

Reply:
A result of this is that the live document website has a self-referential link in the Documentation section.  I kinda think the README should be simpler and link to the full docs or the Getting Started

Reply:
As in
* strip the README largely bare except for maybe the acknowledgements section
* keep a link to the full docs
* remove README from rendered document on website (instead create a separate page with all the details currently in the README

Reply:
Yes, all of that and the README could just say a paragraph of "what bioio does" in addition to a link to docs and maybe acknowledgements or whatever else.   Maybe a very brief getting started usage example of loading image data.

Reply:
Gave it a shot, let me know if you think stuff should be moved around more